pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -444,6 +445,65 @@ func getAddressByName(client *gophercloud.ServiceClient, name types.NodeName) (s
 	return addrs[0].Address, nil
 }
 
+// getInterfacesByName returns the node interfaces of the specified instance.
+func getInterfacesByName(client *gophercloud.ServiceClient, name types.NodeName) ([]attachinterfaces.Interface, error) {
+	var interfaces []attachinterfaces.Interface
+
+	// get server
+	srv, err := getServerByName(client, name)
+	if err != nil {
+		if err == ErrNotFound {
+			return interfaces, cloudprovider.InstanceNotFound
```

Comment:
I also think cloudprovider.InstanceNotFound is better than ErrNotFound.
In order to be easier to review, I will new PR to update ErrNotFound which comes from getServerByName().
Thank for your comment. @zhouhaibing089